detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/baseline/BaselineHandler.kt

```diff
@@ -7,18 +7,20 @@ internal class BaselineHandler : DefaultHandler() {
 
     private var current: String? = null
     private var content: String = ""
-    private val whiteIds = mutableSetOf<String>()
-    private val blackIds = mutableSetOf<String>()
+    private val temporarySuppressedIds = mutableSetOf<String>()
+    private val falsePositiveIds = mutableSetOf<String>()
```

Comment:
Please keep the name of those private fields consistent with the XML names.

```suggestion
    private val manuallySuppressedIssues = mutableSetOf<String>()
    private val currentIssues = mutableSetOf<String>()
```

Reply:
Oops, I thought I had all of them, but missed one. Nice catch!